pkg/sensors/tracing/lsm_test.go

```diff
@@ -236,3 +239,86 @@ spec:
 	err = jsonchecker.JsonTestCheck(t, ec.NewUnorderedEventChecker(lsmChecker))
 	assert.NoError(t, err)
 }
+
+func TestLSMIMAHash(t *testing.T) {
+	if !bpf.HasLSMPrograms() || !kernels.EnableLargeProgs() || !kernels.MinKernelVersion("6.0") {
+		t.Skip()
+	}
```

Comment:
it'd be great to have this tested as part of CI, which I guess means booting with all lsm stuff.. I'll try to check on that

Reply:
For kernels 6+ bpf_ima_file_hash helper works with out IMA, so some vm tests are working already!